packages/docs/source/building-blocks/elements.md

```diff
@@ -0,0 +1,222 @@
+>  - <cite><a href='https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote'>MDN</a></cite>
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <blockquote cite="https://www.huxley.net/bnw/four.html">
+      <p>Words can be like X-rays, if you use them properly—they’ll go through anything. You read and you’re pierced.</p>
+      <footer>—Aldous Huxley, <cite>Brave New World</cite></footer>
+    </blockquote>
+  </div>
+
+```html
+  <blockquote cite="https://www.huxley.net/bnw/four.html">
+    <p>Words can be like X-rays, if you use them properly—they’ll go through anything. You read and you’re pierced.</p>
+    <footer>—Aldous Huxley, <cite>Brave New World</cite></footer>
+  </blockquote>
+```
+</figure>
+
+## cite <a name="cite"></a>
+> The HTML Citation element (`<cite>`) is used to describe a reference to a cited creative work, and must include the title of that work.
+>  - <cite><a href='https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite'>MDN</a></cite>
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <blockquote>
+        <p>It was a bright cold day in April, and the clocks were striking thirteen.</p>
+        <footer>
+            First sentence in <cite><a href="http://www.george-orwell.org/1984/0.html">Nineteen Eighty-Four</a></cite> by George Orwell (Part 1, Chapter 1).
+        </footer>
+    </blockquote>
+  </div>
+
+```html
+  <blockquote>
+    <p>It was a bright cold day in April, and the clocks were striking thirteen.</p>
+    <footer>
+      First sentence in <cite><a href="http://www.george-orwell.org/1984/0.html">Nineteen Eighty-Four</a></cite> by George Orwell (Part 1, Chapter 1).
+    </footer>
+  </blockquote>
+```
+</figure>
+
+## del <a name="del"></a>
+> The HTML `<del>` element represents a range of text that has been deleted from a document. This can be used when rendering "track changes" or source code diff information, for example. The `<ins>` element can be used for the opposite purpose: to indicate text that has been added to the document.
+>  - <cite><a href='https://developer.mozilla.org/en-US/docs/Web/HTML/Element/del'>MDN</a></cite>
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <blockquote>
+        There is <del>nothing</del> <ins>no code</ins> either good or bad, but <del>thinking</del> <ins>running it</ins> makes it so.
+    </blockquote>
+  </div>
+
+```html
+  <blockquote>
+      There is <del>nothing</del> <ins>no code</ins> either good or bad, but <del>thinking</del> <ins>running it</ins> makes it so.
+  </blockquote>
+```
+</figure>
+
+## em <a name="em"></a>
```

Comment:
Can we add examples with multiple levels of emphasis, since it's supported. I'm not sure how we want to visualize that, maybe italic, then semi-bold + italic, then bod + italic?

@joejezowski-okta @nickarcidy-okta  thoughts?